manim/utils/ipython_magic.py

```diff
@@ -137,12 +137,5 @@ def construct(self):
                 )
 
 
-def _video_hash(path):
-    sha1 = hashlib.sha1()
-    with open(path, "rb") as f:
-        while True:
-            data = f.read(65536)
-            if not data:
-                break
-            sha1.update(data)
-    return sha1.hexdigest()
+def _generate_file_name():
+    return config["scene_names"][0] + "@" + datetime.now().strftime("%Y-%m-%d@%H-%M-%S")
```

Comment:
```suggestion
    return config["scene_names"][0] + "@" + datetime.now().strftime("%y-%m-%d@%H-%M-%S")
```
got this from here:
![image](https://user-images.githubusercontent.com/44469195/121228827-84ddbb80-c88d-11eb-8448-f4de091de702.png)

Reply:
@ryanmccauley211 : I suggested the change to two digits here, and then committed the changes directly by accident. Sorry!